src/cli/paths.rs

```diff
@@ -43,7 +44,11 @@ pub(crate) fn proofs_default_dir() -> Utf8PathBuf {
 }
 
 pub(crate) fn commits_default_dir() -> Utf8PathBuf {
-    Utf8PathBuf::from("commits")
+    Utf8PathBuf::from(".lurk/commits")
```

Comment:
What's the rationale for this change? Existing behavior is for `commits` to be located in the local workdir as commitments can contain private data and shouldn't be shared

Reply:
Nvm I was mistaken, it should default to $HOME/.lurk/commits (see DMs)